content/pt-br/docs/concepts/overview/working-with-objects/object-management.md

```diff
@@ -0,0 +1,180 @@
+tipos de recursos cujas especificações são atualizadas independentemente 
+do arquivo de configuração. Serviços do tipo `LoadBalancer`, por exemplo, 
+possuem seu campo `externalIPs` atualizado independente da configuração 
+do cluster.
+{{< /warning >}}
+
+### Exemplos
+
+Crie os objetos definidos no arquivo de configuração: 
+
+```sh
+kubectl create -f nginx.yaml
+```
+
+Exclua os objetos definidos em dois arquivos de cofiguração:
+
+```sh
+kubectl delete -f nginx.yaml -f redis.yaml
+```
+
+Atualize os objetos definidos em um arquivo de configuração substituindo
+a configuração ativa:
+
+```sh
+kubectl replace -f nginx.yaml
+```
+
+### Comparações 
+
+Vantagens comparadas à comandos imperativos:
+
+- A configuração do objeto pode ser salva em um sistema de controle de código, como o Git.
+- A configuração do objeto pode integrar com processos como revisão de mudanças antes de *push* e trilhas de auditoria.
+- A configuração do objeto fornece um *template* para criar novos objetos.
+
+Desvantagens comparadas aos comandos imperativos:
+
+- A configuração do objeto exige um entendimento básico ao esquema de objetos.
+- A configuração do objeto exige um passo adicional para escrever o arquivo YAML.
+
+Vantagens comparadas à configuração da declaração do objeto:
+
+- O comportamento de configuração do objeto imperativo é simples e fácil de entender.
+- A partir da versão 1.5 do Kubernetes a configuração do objeto imperativo é mais madura.
+
+Desvantagens comparadas à configuração da declaração do objeto:
+
+- A configuração do objeto imperativo funciona melhor em arquivos e não diretórios.
+- Atualizações a objetos ativos devem ser mencionadas nos arquivos de configuração ou serão perdidas durante a próxima substituição.
+
+## Configuração da declaração do objeto
+
+Ao usar configuração da declaração de objeto, um usuário opera em arquivos 
+de configuração de objetos armazenados localmente, entretanto o usuário não
+define as operações a serem realizadas nos arquivos. Operações de criar, atualizar
+e deletar são detectadas automaticamente por objeto pelo `kubectl`. Isso hablita
+trabalhar em diretórios, onde operações diferentes podem ser necessárias para
+diferentes objetos.
+
+{{< note >}}
+Configuração da declaração do objeto retém mudanças realizadas por outros
+agentes de escrita, mesmo que as alterações não sejam mescladas de volta 
+ao arquivo de configuração do objeto. Isso é possível usando a operação
+da API `patch` para gravar apenas as diferenças observadas, em vez de usar
+a operação da API `replace` para substituir toda a configuração do objeto. 
+{{< /note >}}
+
+### Exemplos
+
+Processe todos os arquivos de configuração de objetos no diretório `configs` e
+crie ou corrija os objetos ativos. É possível primeiro executar `diff` para ver
+o que mudou e então aplicar:
+
+```sh
+kubectl diff -f configs/
+kubectl apply -f configs/
+```
+
+Diretórios do processo recursivamente: 
```

Comment:
I believe this should have been "processe diretórios recursivamente".

Reply:
Ou processamento dos diretórios recursivamente